src/XIOPAK.H

```diff
@@ -26,26 +26,26 @@ const int XFBUFALLOC=EROP1;	// allocate character buffer in xfOpen()
 	unknown:   File is created if it does not exist but preserved if
 		   it does.  This is most applicable when appending. */
 // Text modes
-#define O_RT (_O_RDONLY|_O_TEXT)					// Read
-#define O_WT (_O_RDWR|_O_TEXT|_O_CREAT|_O_TRUNC)	// r/w (fresh)
-#define O_WTOLD (_O_RDWR|_O_TEXT) 					// r/w (must exist)
-#define O_WTOLDA (_O_RDWR|_O_TEXT|_O_APPEND)		// r/w (append)
-#define O_WTUNKA (_O_WTOLDA|_O_CREAT)				// r/w append, create if nec
+#define O_RT		"r"				// Read
+#define O_WT		"w+"			// r/w (fresh)
+#define O_WTOLD		"r+" 			// r/w (must exist)
+#define O_WTOLDA	"a+"			// r/w (append)
+#define O_WTUNKA	"a+"			// r/w append, create if nec
 
 // Binary
-#define O_RB (_O_RDONLY|_O_BINARY)		     		// Read
-#define O_WB (_O_RDWR|_O_BINARY|_O_CREAT|_O_TRUNC)	// r/w (fresh)
-#define O_WBOLD (_O_RDWR|(_O_BINARY) 				// r/w (old, must exist)
-#define O_WBOLDA (O_WBOLD|_O_APPEND)	  			// r/w append (must exist)
-#define O_WBUNK (O_WBOLD|_O_CREAT)	  				// r/w (create if nec.)
-#define O_WBUNKA (O_WBOLDA|_O_CREAT)	  			// r/w append (create if nec.)
+#define O_RB		"rb"			// Read
+#define O_WB		"wb+"			// r/w (fresh)
+#define O_WBOLD		"rb+"			// r/w (old, must exist)
+#define O_WBOLDA	"ab+"			// r/w append (must exist)
+#define O_WBUNK		"rb+"			// r/w (create if nec.)
+#define O_WBUNKA	"ab+"			// r/w append (create if nec.)
 
```

Comment:
How do we feel about adding a few definitions here to correspond to the _open function's _oflags_ (_O_APPEND, _O_WRONLY, _O_BINARY etc). I'm seeing a lot of literals in the new fopen calls like "ab" and "wb" which might be better to consolidate in one place. @nealkruis?